Azure/communication-ui-library-ios
# [Bugfix] Show info header with keyboard

## Purpose

* Show info header when keyboard focus change is detected for accessibility purposes

## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[X] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code


[Enable full keyboard access](https://www.iphonelife.com/content/ipad-keyboard-shortcuts-to-navigate-apps)

1. Open Application using space key
2. Application will be open, navigate to Swift UI button and activate it using space key.
3. Navigate to ACS Token Button and activate it using space key
4. Fill out required details like ACS token in 'ACS Token' test field and name in 'display name' text field.
5. Navigate to 'Team's Meeting' text field and enter their team's meeting link.
6. Navigate to 'Start Experience' button.
7. Call setup screen will appear
8. Navigate to 'Join Call' button using tab key and activate it using space key.
9. Meeting will be started after joining by Team's member.
10. Info Header of Participant will appear.
11. Info Header will be hide after few second.
12. Press the tab and arrow keys to change focus and to bring up info header.

## What to Check
Verify that the following are valid
* ...

## Other Information
